bin/build-docker-images.sh

```diff
@@ -4,3 +4,6 @@ docker-compose build tomcat_oscar
 docker tag open-osp_tomcat_oscar openosp-oscar:latest
 docker push openosp-oscar:latest
 
+docker-compose build tomcat_expedius
```

Comment:
I'm guessing we might want to sometimes build expedius only, or oscar only, so probably a flag, switch or separate command to separate those optionally?